pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -201,12 +266,93 @@ func doCreateBackupSchedule(
 	if err != nil {
 		return errors.Wrapf(err, "failed to evaluate backup destination paths")
 	}
+
+	if len(destinations) > 1 {
+		if !eval.isEnterpriseUser {
+			if err := utilccl.CheckEnterpriseEnabled(
+				p.ExecCfg().Settings, p.ExecCfg().ClusterID(), p.ExecCfg().Organization(),
+				"partitioned destinations"); err != nil {
+				return err
+			}
+		}
+	}
 	for _, dest := range destinations {
 		backupNode.To = append(backupNode.To, tree.NewDString(dest))
 	}
 
 	backupNode.Targets = eval.Targets
 
+	var scheduleName string
+	if eval.scheduleName != nil {
+		scheduleName, err = eval.scheduleName()
+		if err != nil {
+			return err
+		}
+	} else {
+		scheduleName = fmt.Sprintf("BACKUP %d", env.Now().Unix())
+	}
+
+	scheduleOptions, err := eval.scheduleOpts()
+	if err != nil {
+		return err
+	}
+
+	return p.ExecCfg().DB.Txn(ctx, func(ctx context.Context, txn *kv.Txn) error {
+		// Create FULL backup schedule.
+		if err := createBackupSchedule(
+			ctx, p, env, scheduleName+": FULL", fullRecurrence,
```

Comment:
@mwang1026 can chime in here too but my vote was we give at least one of the two schedules the actual supplied schedule name.

I'd vaguely say it would be the inc schedule that you'll actually see running most of the time, but I could also see it being the full schedule that _all_ schedules will have for consistency. Indeed if there is only one schedule, it feels weird that it doesn't have the specified name?